target/linux/qualcommax/ipq60xx/base-files/etc/init.d/bootcount

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh /etc/rc.common
+
+START=99
+
+. /lib/functions.sh
+
+boot() {
+        case $(board_name) in
+                yuncore,fap650)
+                        (
+                        # Using brackets and & to prevent block on sleep
+                        echo resetting owrt_bootcount in 5 min...
+                        sleep 300 && fw_setenv owrt_bootcount 0
```

Comment:
Why do you need 5 minute wait to reset the bootcount U-Boot env variable?

Reply:
Just to make sure that device is working good enough. There is a chance to get kernel panic in first two minutes after boot was finished.

Reply:
Yeah, that wont fly here, what is triggering the panic?

That needs to be fixed instead of workarounds like this.

Reply:
Now everything is fine, there is no kernel panic. The kernel starts stably.
There were situations when the next build almost worked. The kernel started and managed to switch the environment variable, but then the device hung. So, I added a 5-minute delay.

Reply:
Then remove it